ParkingFinder/base/errors.py

```diff
@@ -0,0 +1,24 @@
+class BaseError(Exception):
+    pass
+
+
+class AuthorizationError(BaseError):
+    """
+    Invalid Access Token Exception
+
+    """
+    error = 'Invalid Access Token'
+
+
+class NotFound(BaseError):
```

Comment:
I change AuthorizationError to Unauthorized. all internal generic errors in our project are naming without 'Error' as ending in order to distinguish from system error.